cmd/alias-set.go

```diff
@@ -100,19 +101,40 @@ EXAMPLES:
 }
 
 // checkAliasSetSyntax - verifies input arguments to 'alias set'.
-func checkAliasSetSyntax(ctx *cli.Context, accessKey string, secretKey string, deprecated bool) {
-	args := ctx.Args()
+func checkAliasSetSyntax(cliCtx *cli.Context, accessKey string, secretKey string, deprecated bool) (usageList bool) {
+	args := cliCtx.Args()
 	argsNr := len(args)
-	if argsNr > 4 || argsNr < 2 {
-		fatalIf(errInvalidArgument().Trace(ctx.Args().Tail()...),
+
+	if argsNr == 0 {
+		fmt.Println("Use this command with the following arguments to add an alias: mc alias set ALIAS URL ACCESSKEY SECRETKEY. ")
+		fmt.Println("Configured aliases are:")
+		conf, err := loadMcConfig()
+		fatalIf(err.Trace(globalMCConfigVersion), "Unable to load config version `"+globalMCConfigVersion+"`.")
+
+		for k := range conf.Aliases {
+			// Create a new MinIO Admin Client
+			client, err := newAdminClient(k)
+			fatalIf(err, "Unable to initialize admin connection.")
+			admInfo, e := client.ServerInfo(globalContext)
+			// this is not the right way to handle this error - find the correct way
+			if e != nil {
+				fmt.Println("Unable to retrieve usage information for `" + k + "`.")
+			} else {
+				fmt.Println("Usage for `" + k + "` is `" + humanize.IBytes(admInfo.Usage.Size) + "`.")
+			}
```

Comment:
If there are configured endpoints that are not reachable this command will never return @jinapurapu - I don't think this is a feasible behavior.